packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx

```diff
@@ -110,11 +116,11 @@ function CreatePinPage(props: CreatePinPageProps) {
     const doRegister = async (hashId: string, publicKey: string) => {
         try {
             setLoading(true)
-            const {vaddrs, vopts} = await Services.Eligibility.register(hashId, publicKey);
-            setVotingOptions(vopts);
-            setVotingAddress(vaddrs);
+            await Services.Eligibility.register(hashId, publicKey);
+            setCanSetPin(true);
```

Comment:
why not `setIsElgible(true)`

Reply:
because isEligeble is already true when component starts

Reply:
and it is set to false when request fails, at the catch(e)

Reply:
PR Updated to use `IS_ELIGIBLE_ENUM`